Unity-Technologies/com.unity.multiplayer.samples.bitesize
# Fix: Client Driven Ghost Static Ingredients on Late join [MTT-4505]

### Description

To remove the inconsistent behavior for late joiners (ingredients existing on late joiners side even though they were removed by another player) the in-scene placed objects got removed and spawners similar to the ones in Boss Room got added to achieve consistent behavior between clients 

How to test:

1. Open Client Driven Sample
2. Create a build or clone
3. Join as host and bring one ingredient to goal with host
4. Late-join from other build/editor and check that scene looks identical for both players

### Issue Number(s)


[MTT-4505](https://jira.unity3d.com/browse/MTT-4505)

### Contribution checklist
